app/controllers/puppetclasses_controller.rb

```diff
@@ -85,4 +86,8 @@ def action_permission
         super
     end
   end
+
+  def fetch_associations
+    ActiveRecord::Associations::Preloader.new(@puppetclass.lookup_keys, [:lookup_values]).run
```

Comment:
Can we use .includes with find_resource instead?  This class doesn't appear to be part of the intended public API (https://github.com/rails/rails/issues/11719#issuecomment-22006154).